src/corelibs/U2View/U2View.pro

```diff
@@ -96,6 +96,7 @@ HEADERS += src/LicenseDialog.h \
            src/ov_msa/MsaOpSavableTab.h \
            src/ov_msa/MSASelectSubalignmentDialog.h \
            src/ov_msa/MsaEditorUserModStepController.h \
+           src/ov_msa/AlignSequencesToAlignment/AlignSequencesToAlignmentSupport.h \
```

Comment:
We usually name the inner directories is a "snake_case" style. Like `align_sequences_to_alignment`

Reply:
I agree, but I have not added any folders here + there are 10 other subfolders on the same level to be fixed.
Will fix them all in the separate PR.